datahub-web-react/src/app/ingest/source/utils.ts

```diff
@@ -40,7 +43,10 @@ export function getPlaceholderRecipe(ingestionSources: SourceConfig[], type?: st
 
 export const RUNNING = 'RUNNING';
 export const SUCCESS = 'SUCCESS';
+export const FLAGGED = 'FLAGGED';
+export const WARNING = 'WARNING';
 export const FAILURE = 'FAILURE';
+export const CONNECTION_FAILURE = 'CONNECTION_FAILURE';
```

Comment:
really happy to finally see us adding more rich statuses.

we are doing the same in datahub-monitors-service when running a test. we should align on the error types where possible.